### DISCUSSION TITLE
fixedwing in Airsim

### DISCUSSION THREAD

**Discussion by airsimdevdd** [microsoft/AirSim#4202] [Support Q&A]
Fixed wing in Airsim , Any Modified version of Airsim  to work with fixed wing please help   @AOS55 
@rajat2004 @jonyMarino  @seanmcleod  @xxEoD2242  @nikitabeebe

**Comment by zimmy87** [comment]
There is a python module for passing JSBSim output to AirSim. Currently supports two fixed wing aircraft, a C172 and an x8 UAV. The repo for this module can be found here: https://github.com/AOS55/Fixedwing-Airsim

**Reply by airsimdevdd** [reply]
@zimmy87 but can i connect autopilot like px4 and also 
need physical cad model of the fixed wing
